index.html

```diff
@@ -25,21 +25,178 @@
+                    <img src="image/works4.png" alt="Avocode Snapshoot">                    
+                </figure>
+                <div class="work-desc">
+                    <h2>Multi-Post
+                        Stories</h2>
+                    <div class="sub-box">
+                        <h3>CANOPY</h3>
+                        <ul>
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                        </ul>                        
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="small-button">
+                        <li>hmtl</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <button type="button" class="see-button">See Project</button>
+                </div>
+            </div>
+            <!-- fin trabajo 4 -->
+
+
+        </section>
+
+        <!-- Seccion about -->
+        <section name="about-myself" class="about-myself">
+            <div class="about-desc">
+                <h2>About <br>Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <h3 class="sub-txt">LET'S CONNECT</h2>
+                <ul class="social brd-btm">
+                    <li><a href="#"><img src="image/ico-twitter.png" alt="Twitter"></a></li>
+                    <li><a href="#"><img src="image/ico-likedin.png" alt="linkedin"></a></li>
+                    <li><a href="#"><img src="image/ico-M.png" alt="Microverse"></a></li>
+                    <li><a href="#"><img src="image/ico-hithub.png" alt="HitHub"></a></li>
+                    <li><a href="#"><img src="image/ico-hand.png" alt="Peace ico"></a></li>
+                </ul>
+                <button type="button" class="see-button">Get my resume</button>
+            </div>
+            <div class="about-skills">
+                <ul>
+                    <li>
+                        <div class="languages spc-36">
+                            <span>Languages</span><img src="image/down-ico.png" alt="Arrow down">                            
+                        </div>
+                    </li>
+                        <div class="lng-item">
+                            <img src="image/js-logo.png" alt="JS logo"><span>JavaScript</span>
+                        </div>
+                        <div class="lng-item">
+                            <img src="image/html-logo.png" alt="HTML logo"><span>HTML</span>
+                        </div>
+                        <div class="lng-item">
+                            <img src="image/css-logo.png" alt="CSS logo"><span>CSS</span>
+                        </div>
```

Comment:
- [ ] The list of languages works like a list and they would better fit inside the `li` of the languages. Every item inside a `ul` should be wrapped with the `li` tag for better semantics so please move the 'javascript', 'html' and 'css' items to inside the `li` of the languages above it.

Reply:
I move the 'JavaScript', 'HTML' and 'CSS' items to inside the li of the languages above it.
![image](https://user-images.githubusercontent.com/110146573/221306719-7e56c442-bb95-473f-9a1b-142590f287cd.png)